.github/workflows/csv-lint.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         run: |
           cd csvlint
           for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
```

Comment:
> Is 42 subject to injection?

I don't believe so.  My understanding is that bash has a strict interpenetration of the input parameter for a `for` loop.  Similar to how command injection should not be possible with `FOO=${{ steps.changed-files.outputs.all_changed_files }}`

Related since we are discussing it, I considered also setting the `IFS` to correctly handle filenames with spaces.  But setting this value would actually make a theoretically command injection easier (though in theory not possible given the `printf` change).

Let me know if you have any opinion on if I should make that change.

> Did you asses env laundering as a potential mitigation?

As I understand it the `printf` would still be necessary. Once the `for` loop logic breaks apart the input I think injection may still be possible.  Do you think it's worth adding this protection given the above?  I have no concerns, just trying to keep complexity to a minimum.

Reply:
@wadells friendly ping on this when you have a chance.  Not high priority, but it would be nice to finish this NCC finding out.

Reply:
Sorry -- you hadn't requested re-review, so this wasn't showing up in my review queue:

![image](https://github.com/gravitational/shared-workflows/assets/187314/2e262fab-5375-4adf-9a58-576f6f9ce577)

The best way to make sure I (or most other folks) revisit something (especially if I've requested changes) is to re-request review.

Reply:
Thank you, requested.  To be clear I have not made any changes yet, I am just looking for feedback on if we want to additionally add the environment filter.